### DISCUSSION TITLE
JSON-LD Support

### DISCUSSION THREAD

**Discussion by DMiradakis** [unjs/unhead#157] [Ideas]
Does Unhead currently support JSON-LD? I can't seem to find references to it in the docs.

**Comment by harlan-zw** [comment]
Yes, it does, just use the correct type with innerHTML, you can even provide a plain object

```ts
useHead({
    script: [
      {
        type: 'application/ld+json',
        innerHTML: '<schema>',
      },
    ],
  })
````

I'd suggest using https://unhead-schema-org.harlanzw.com/ though

**Reply by DMiradakis** [reply]
That's helpful, thanks. 😃
